frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -515,7 +516,7 @@ const DeviceUserPage = ({
                       deviceToken={deviceAuthToken}
                       isSoftwareEnabled
                       pathname={location.pathname}
-                      queryParams={parseHostSoftwareQueryParams(location.query)}
```

Comment:
This was throwing a bunch of unnecessary host software query params into the URL bar for self service page